src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -125,6 +128,10 @@ public boolean isFailOnUnknownTokens() {
     return failOnUnknownTokens;
   }
 
+  public boolean isEnableNestedInterpretation() {
```

Comment:
There is also one enableRecursiveMacroCalls, do you think we should rename them together?